libs/features/mas/src/mas-commerce-service.js

```diff
@@ -143,8 +146,16 @@ export class MasCommerceService extends HTMLElement {
                 detail: this,
             });
             performance.mark(MARK_READY);
+            this.initDuration = performance.measure(
+                Constants.MAS_COMMERCE_SERVICE_INIT_TIME_MEASURE_NAME,
+                MARK_START,
+                MARK_READY,
+            )?.duration;
             this.dispatchEvent(event);
         });
+        setTimeout(() => {
+            this.logFailedRequests();
+        }, 10000);
```

Comment:
can we make this a DEFAULT_TIMEOUT in constants?

Reply:
@npeltier it is not a timeout, it is just to wait 10s after the initialization to send the logs.